### PR TITLE
Refs unification

### PR DESCRIPTION
**Pull request by alirezazolanvari**
#### Reference Issues/PRs
#173 

#### What does this implement/fix? Explain your changes.
The references are unified with IEEE format

#### Any other comments?
Please carefully compare the new references' titles with the previous ones. Be sure that the formats are correct.

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@alirezazolanvari Thanks for your efforts 💯 
I will review this PR in the next few days.
But as a side note, I think it would better if we mention `IEEE` format somewhere in `CONTRIBUTING.md`

File: README.md
Comment (sadrasabouri) [inline README.md:809]: another blank space here :point_up:

File: README.md
Comment (sadrasabouri) [inline README.md:813]: another blank space here :point_up:

File: README.md
Comment (sadrasabouri) [inline README.md:815]: another blank space here :point_up:

File: README.md
Comment (sadrasabouri) [inline README.md:819]: another white space here :point_up:

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thank you @alirezazolanvari :1st_place_medal: 
There were tiny things that I mentioned in comments, also there were some changes in date of each references that I decided not to mention them (due to make the review lighter) but there should be reasonable reason why these changes are necessary.

**Comment by sadrasabouri** [comment]
> @alirezazolanvari Thanks for your efforts 100
> I will review this PR in the next few days.
> But as a side note, I think it would better if we mention `IEEE` format somewhere in `CONTRIBUTING.md`

Also logging this change in `CHANGELOG.md` may be helpful.

File: README.md
Comment (alirezazolanvari) [reply README.md:777]: In IEEE format for citing conference papers, `in` should be written before the conference's name.

**Review by sadrasabouri — APPROVED** [review]

File: README.md
Comment (sepandhaghighi) [inline README.md:849]: `conferenceon`?

File: README.md
Comment (sepandhaghighi) [inline README.md:853]: Journal name is `Journal of statistical software`

https://www.jstatsoft.org/article/view/v028i05

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sepandhaghighi** [comment]
@alirezazolanvari 
Please take a look at my comments, then I will update `CHANGELOG.md`, `CONTRIBUTING.md` and `Document.ipynb`

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
LGTM 👍 
@sadrasabouri @alirezazolanvari Please take a look at my commits (https://github.com/sepandhaghighi/pycm/pull/362/commits/0bf28f46cec4599659e624c5074e57bd2de5090a, https://github.com/sepandhaghighi/pycm/pull/362/commits/cd416c1f14c8432aeb957b149d0d5be204e80b7c, https://github.com/sepandhaghighi/pycm/pull/362/commits/ad546700702c24906f4e1dee03133bfbaaa2fc00)
If everything is okay, please merge this `PR` after approval.

**Review by sadrasabouri — APPROVED** [review]
LGTM

**Pull request merged**
